ccd-definition/CaseEvent.json

```diff
@@ -3,7 +3,7 @@
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "ID": "openCase",
```

Comment:
I think it might be good idea to change event ID as well to `startApplication`. Before doing so please check how it impacts existing cases.

1. create new case using `openCase` event
2. rename event and upload new definition
3. check whether you can still see case crated in #1 and that you can see full history.

Reply:
My prediction was right. You did not notice that "Open case" event disappeared from event history of case that was created before rename :( Please be more vigilant next time, it is in developer hands to ensure backward compatibility.

Reply:
Please revert event ID rename due to its price :(

Reply:
@rebeccabrennan since you have issues with machine I carried out that change for you.